runtime/src/mm/cpp/Memory.cpp

```diff
@@ -263,6 +263,48 @@ extern "C" void Kotlin_native_internal_GC_collectCyclic(ObjHeader*) {
     ThrowIllegalArgumentException();
 }
 
+extern "C" void Kotlin_native_internal_GC_setThreshold(ObjHeader*, int32_t value) {
```

Comment:
Reusing existing `kotlin.native.internal.GC` API for tuning GC. It's a temporary solution.